api/pkg/s3/datastore/ceph/ceph.go

```diff
@@ -141,6 +141,10 @@ func (ad *CephAdapter) DELETE(object *pb.DeleteObjectInput, ctx context.Context)
 	return NoError
 }
 
+func (ad *CephAdapter) POST(object *pb.RestoreObjectInput, ctx context.Context) S3Error {
+	return NoError
+}
```

Comment:
it's better return NoImplementedError

Reply:
I think NoError is okay, NoImplementedError doesn't look meaningful